src/corehost/cli/hostpolicy/run_context.h

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __RUN_CONTEXT_H__
+#define __RUN_CONTEXT_H__
+
+#include <pal.h>
+
+#include "args.h"
+#include "coreclr.h"
+#include "hostpolicy_init.h"
+
+struct run_context_t
```

Comment:
The more I look at this, the less I like this name... "run" is not a good term here.
I would call it "host context", but then we would have a name collision with `hostfxr` which will also have a "host context".
So some ideas:
* `hostpolicy_context` - would make it pretty easy to understand and would nicely differentiate from `hostfxr`. Downside is that we don't really use `hostpolicy_` prefix anywhere.
* `corehost_context` - aligns with the rest of the `corehost_` stuff, but collides with the proposed public `corehost_context_contract` at least a little bit.

I think I would prefer the `hostpolicy_context` - the `corehost_` prefix is really weird, so let's not expand it further (outside of the public APIs).

Reply:
Yeah, I renamed this a couple of times and was not really happy with anything...
I prefer `hostpolicy_context` as well - will switch to that.